src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    Circle(double radius, Color color) {

```

Comment:
```suggestion
    public Circle(double radius, Color color) {
```
the same for other figures